.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Update the Code Coverage
+
+on:
+  workflow_dispatch
```

Comment:
This workflow should be triggered upon pushing to the main branch only and should not run whenever the release workflow pushes that release version update in the POM file if you have that step of pushing POM file change in the release script.

Reply:
If the main branch's last commit message contains `ci skip` word then this workflow will not run.  And In the release script, I will add this  `ci skip`  keyword in the commit message of POM changes.